cmd/controlplane/controller.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/akuity/kargo/internal/controller/stages"
 	"github.com/akuity/kargo/internal/controller/warehouses"
 	"github.com/akuity/kargo/internal/credentials"
+	_ "github.com/akuity/kargo/internal/gitprovider/github"
```

Comment:
I am using the technique of anonymous imports so that providers can "register" themselves through their respective init() functions. This avoids an import cycle, or the need for an extra instantiation package that knows about all providers.